packages/elements/src/components/ino-snackbar/ino-snackbar.scss

```diff
@@ -138,6 +126,10 @@ ino-snackbar {
       color: #74758b;
     }
 
+    .ino-snackbar-action-container {
+      margin-top: 0.5rem;
```

Comment:
IMO a smaller margin (e.g. `0.25rem`) would fit a lot better. What do you think? @BenPag 

![image](https://github.com/inovex/elements/assets/22963121/8503f899-1ffc-4640-aa66-5c83d1bfa893)

```suggestion
      margin-top: 0.25rem;
```

Reply:
I agree, a smaller margin fits better.

Reply:
also fix alignment of X button